index.html

```diff
@@ -0,0 +1,122 @@
+           
+            <compactbar class="compactbar">
+                <toobardark class="toobardark">
+                   
+                    <normalbutton class="normalbutton">
+                        <icons class="icons">
+
+                            <unions class="unions">
+                                <img src="img/Vector.png" alt="Network">
+                                <vector1 class="vector1">
+
+                                </vector1>
+                                <vector2 class="vector2">
+                                    
+                                </vector2>
+                                <vector3 class="vector3">
+                                    
+                                </vector3>
+                            </unions>
+
+                        </icons>
+
+                    </normalbutton>
+
+                </toobardark>
+                <logoplaceholder class ="logoplaceholder">
+                    <img src="img/logoplaceholder.png" alt="Network"> 
+
+                </logoplaceholder>
+
+            </compactbar>
+        </nav>  
+      
+        
+        <main class=" backgroundimg">
+            <goupbelowmenu class="groupbelowmenu">
+                
+                <img src="img/vectorbelowmenu.png" alt="Network">  
+            </goupbelowmenu>
+            <primaryinmain class="primaryinmain">
+                <h1> 
+                    Hey there.</h1>
+
+                   <h1>im Samuel.</h1>  
+
+                   <h1> I’m a software 
+                    
+                    developer <h1>
+                
+            </primaryinmain>
+            <supportingtext class="Supportingtext">
+                <contect>  I can help you build a product , feature or website
+                     Look through some of my work and experience! If you like what you 
+                     see and have a project you need coded, 
+                    don’t hesitate to contact me.
+                </contect> 
+
+            </supportingtext>
+            <br>
+            <br>
+            <socialmedia class="socialmedia">
+                <twitter class="twitter">
+                    <img src="img/twitter.png" alt="Network">  
+                </twitter>
+                <vectori class="vectori">
+                    <img src="img/vectori.png" alt="Network">  
+                </vectori>
+                <vectorn class="vectorin">
+                    <img src="img/vectorn.png" alt="Network">  
+                </vectorn>
+                <medium class="medium">
+                    <img src="img/medium.png" alt="Network">  
+                </medium>
+                <github class="github">
+                    <img src="img/github.png" alt="Network">  
+                </github>
+                <angellist class="angellist">
+                    <img src="img/angellist.png" alt="Network">  
+                </angellist>
+
+            </socialmedia>
```

Comment:
- [ ] [REQUIRED] Good job so far! The list of icons should use a `ul` tag to group them, and `li` per icon; each icon should have a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.